### ISSUE TITLE
Minor visual differences with production App Inventor

### ISSUE THREAD

**Issue by ewpatton** [mit-cml/blockly-plugins#51]
Note that removing the CSS style here causes the black text on light orange that we see in production App Inventor becomes white on light orange, which violates WCAG contrast guidelines (drops from 8.25:1 to 2.54:1)

https://github.com/mit-cml/blockly-plugins/blob/1c0db82346cd8910484f821590d1c4f31f7c67be/block-lexical-variables/src/fields/field_flydown.js#L91-L92

Contrast checker: https://webaim.org/resources/contrastchecker/
Background color: DE8F6C

I'd recommend that we leave the CSS styling.

**Comment by mark-friedman** [comment]
#52 improves contrast by using black text, but it doesn't make the colors be the same as App Inventor's.  I've done a bunch of investigating the CSS with the Chrome devtools and following code paths in the plugin and in App Inventor, but I've hit a wall.
